RELEASE.md

```diff
@@ -32,3 +32,16 @@ Ideally, if the bundle has to be updated in mozilla-central for a bugfix, a corr
 tag should be created on GitHub.
 
 [1] https://bugzilla.mozilla.org/enter_bug.cgi?product=Firefox&component=Developer%20Tools%3A%20Shared%20Components
+
+
+### How to publish a new Reps release to npm.
+
+Reps can be installed by application through npm ([see package page](https://www.npmjs.com/package/devtools-reps)).
```

Comment:
I don't understand this paragraph, do you mean that "Reps can be installed by any application using the npm tool"? or that any application can install and use reps? Either way, it feels somehow superfluous in the context of how to make a release. I'd suggest you remove the whole sentence altogether :)